# HG changeset patch
# User Artem Krosheninnikov <artem.krosheninnikov@gmail.com>
# Date 1383428564 25200
#      Sat Nov 02 14:42:44 2013 -0700
# Node ID 0dc5a1b3a4db4b9036eb56ef1a08f4f30225a0b7
# Parent  a5c4efa2e5a3bf5d6c9fb67d1e3fe37a2f03ac46
ru_RU.ts: Update Octave Russian translation.

* libgui/languages/ru_RU.ts: Update Octave Russian translation.

* doc/interpreter/contributors.in: Add Artem Krosheninnikov to list of
contributors.

diff --git a/doc/interpreter/contributors.in b/doc/interpreter/contributors.in
--- a/doc/interpreter/contributors.in
+++ b/doc/interpreter/contributors.in
@@ -133,16 +133,17 @@ Arno J. Klaassen
 Alexander Klein
 Geoffrey Knauth
 Heine Kolltveit
 Ken Kouno
 Kacper Kowalik
 Daniel Kraft
 Aravindh Krishnamoorthy
 Oyvind Kristiansen
+Artem Krosheninnikov
 Piotr Krzyzanowski
 Volker Kuhlmann
 Tetsuro Kurita
 Philipp Kutin
 Miroslaw Kwasniak
 Rafael Laboissiere
 Kai Labusch
 Claude Lacoursiere
diff --git a/libgui/languages/ru_RU.ts b/libgui/languages/ru_RU.ts
--- a/libgui/languages/ru_RU.ts
+++ b/libgui/languages/ru_RU.ts
@@ -1,17 +1,17 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!DOCTYPE TS>
 <TS version="2.0" language="ru_RU">
 <context>
     <name>ListDialog</name>
     <message>
         <location filename="../src/dialog.cc" line="+250"/>
         <source>Select All</source>
-        <translation type="unfinished"></translation>
+        <translation>Выбрать всё</translation>
     </message>
 </context>
 <context>
     <name>QObject</name>
     <message>
         <location filename="../src/workspace-model.cc" line="+75"/>
         <source>automatic</source>
         <translation type="unfinished"></translation>
@@ -62,640 +62,658 @@
         <translation type="unfinished"></translation>
     </message>
 </context>
 <context>
     <name>QTerminal</name>
     <message>
         <location filename="../qterminal/libqterminal/QTerminal.h" line="+116"/>
         <source>Copy</source>
-        <translation type="unfinished"></translation>
+        <translation>Копировать</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Paste</source>
-        <translation type="unfinished"></translation>
+        <translation>Вставить</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Clear All</source>
-        <translation type="unfinished"></translation>
+        <translation>Очистить всё</translation>
     </message>
 </context>
 <context>
     <name>QWinTerminalImpl</name>
     <message>
         <location filename="../qterminal/libqterminal/win32/QWinTerminalImpl.cpp" line="+1527"/>
         <source>copied selection to clipboard</source>
-        <translation type="unfinished"></translation>
+        <translation>выделенное скопировано в буфер обмена</translation>
     </message>
 </context>
 <context>
     <name>documentation_dock_widget</name>
     <message>
         <location filename="../src/documentation-dock-widget.cc" line="+34"/>
         <source>Documentation</source>
-        <translation type="unfinished">Документация</translation>
+        <translation>Документация</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>See the documentation for help.</source>
-        <translation type="unfinished"></translation>
+        <translation>Для получения помощи смотрите документацию.</translation>
     </message>
 </context>
 <context>
     <name>file_editor</name>
     <message>
         <location filename="../src/m-editor/file-editor.cc" line="+304"/>
         <location line="+21"/>
         <location line="+17"/>
         <location line="+57"/>
         <location line="+28"/>
         <source>Octave Editor</source>
-        <translation type="unfinished"></translation>
+        <translation>Редактор Octave</translation>
     </message>
     <message>
         <location line="-249"/>
         <source>Octave Files (*.m);;All Files (*)</source>
-        <translation type="unfinished"></translation>
+        <translation>Файлы Octave (*.m);;Все файлы (*)</translation>
     </message>
     <message>
         <location line="+222"/>
         <source>File not saved! A file with the selected name
 %1
 is already open in the editor</source>
-        <translation type="unfinished"></translation>
+        <translation>Файл не сохранен! Файл с выбранным именем
+%1
+уже открыт в редакторе</translation>
     </message>
     <message>
         <location line="+28"/>
         <source>The associated file editor tab has disappeared.  It was likely closed by some means.</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+228"/>
         <source>&amp;%1 %2</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;%1 %2</translation>
     </message>
     <message>
         <location line="+173"/>
         <source>&amp;New File</source>
-        <translation type="unfinished">Созд&amp;ать</translation>
+        <translation>Созд&amp;ать</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>&amp;Open File</source>
-        <translation type="unfinished">&amp;Открыть</translation>
+        <translation>&amp;Открыть</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>&amp;Save File</source>
-        <translation type="unfinished">&amp;Сохранить</translation>
+        <translation>&amp;Сохранить</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Save File &amp;As</source>
-        <translation type="unfinished">Сохранить &amp;как</translation>
+        <translation>Сохранить &amp;как</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Print</source>
-        <translation type="unfinished"></translation>
+        <translation>Печать</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>&amp;Undo</source>
-        <translation type="unfinished">О&amp;тменить</translation>
+        <translation>О&amp;тменить</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>&amp;Redo</source>
-        <translation type="unfinished">&amp;Повторить</translation>
+        <translation>&amp;Повторить</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>&amp;Copy</source>
-        <translation type="unfinished">&amp;Копировать</translation>
+        <translation>&amp;Копировать</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Cu&amp;t</source>
-        <translation type="unfinished">Вы&amp;резать</translation>
+        <translation>Вы&amp;резать</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Paste</source>
-        <translation type="unfinished"></translation>
+        <translation>Вставить</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>&amp;Next Bookmark</source>
-        <translation type="unfinished">С&amp;ледующая закладка</translation>
+        <translation>С&amp;ледующая закладка</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Pre&amp;vious Bookmark</source>
-        <translation type="unfinished">Пр&amp;едыдущая закладка</translation>
+        <translation>Пр&amp;едыдущая закладка</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Toggle &amp;Bookmark</source>
-        <translation type="unfinished">&amp;Установить/снять закладку</translation>
+        <translation>&amp;Установить/снять закладку</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>&amp;Remove All Bookmarks</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Удалить все закладки</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>&amp;Next breakpoint</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Следующая точка останова</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Pre&amp;vious breakpoint</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Предыдущая точка останова</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Toggle &amp;breakpoint</source>
-        <translation type="unfinished"></translation>
+        <translation>Установить/снять &amp;точку останова</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>&amp;Remove All breakpoints</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Удалить все точки останова</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>&amp;Comment</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Закомментировать</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>&amp;Uncomment</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Раскомментировать</translation>
     </message>
     <message>
         <location line="+63"/>
         <source>&amp;Recent Editor Files</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Недавние файлы редактора</translation>
     </message>
     <message>
         <location line="+9"/>
         <source>&amp;Edit Function</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Редактировать функцию</translation>
     </message>
     <message>
         <location line="+10"/>
         <source>&amp;Close</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Закрыть</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Close All</source>
-        <translation type="unfinished"></translation>
+        <translation>Закрыть всё</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Close Other Files</source>
-        <translation type="unfinished"></translation>
+        <translation>Закрыть другие файлы</translation>
     </message>
     <message>
         <location line="+49"/>
         <source>Run &amp;Selection</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+5"/>
         <source>&amp;Help</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Помощь</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>&amp;Help on Keyword</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Помощь по зарезервированному слову</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>&amp;Documentation on Keyword</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Документация по зарезервированному слову</translation>
     </message>
     <message>
         <location line="-147"/>
         <source>&amp;Find and Replace</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Найти и заменить</translation>
     </message>
     <message>
         <location line="-584"/>
         <source>Could not open file
 %1
 for read: %2.</source>
-        <translation type="unfinished"></translation>
+        <translation>Не удалось открыть файл
+%1
+для чтения: %2.</translation>
     </message>
     <message>
         <location line="+21"/>
         <source>File
 %1
 does not exist. Do you want to create it?</source>
-        <translation type="unfinished"></translation>
+        <translation>Файл
+%1
+не существует. Вы хотите создать его?</translation>
     </message>
     <message>
         <location line="+17"/>
         <source>Could not open file
 %1
 for write: %2.</source>
-        <translation type="unfinished"></translation>
+        <translation>Не удалось открыть файл
+%1
+для записи: %2.</translation>
     </message>
     <message>
         <location line="+549"/>
         <source>Save File And Run</source>
-        <translation type="unfinished"></translation>
+        <translation>Сохранить и запустить файл</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Go&amp;to Line</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Перейти к строчке</translation>
     </message>
     <message>
         <location line="+53"/>
         <source>&amp;File</source>
-        <translation type="unfinished">&amp;Файл</translation>
+        <translation>&amp;Файл</translation>
     </message>
     <message>
         <location line="+39"/>
         <source>&amp;Edit</source>
-        <translation type="unfinished">&amp;Правка</translation>
+        <translation>&amp;Правка</translation>
     </message>
     <message>
         <location line="+22"/>
         <source>&amp;Preferences</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Настройки</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>&amp;Styles Preferences</source>
-        <translation type="unfinished"></translation>
+        <translation>Настройки &amp;стиля</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>&amp;Debug</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Отладка</translation>
     </message>
     <message>
         <location line="+9"/>
         <source>&amp;Run</source>
-        <translation type="unfinished">&amp;Запуск</translation>
+        <translation>&amp;Запуск</translation>
     </message>
 </context>
 <context>
     <name>file_editor_tab</name>
     <message>
         <location filename="../src/m-editor/file-editor-tab.cc" line="+755"/>
         <source>Goto line</source>
-        <translation type="unfinished"></translation>
+        <translation>Перейти к строчке</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Line number</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+70"/>
         <source>&lt;unnamed&gt;</source>
-        <translation type="unfinished"></translation>
+        <translation>&lt;без названия&gt;</translation>
     </message>
     <message>
         <location line="+40"/>
         <source>Do you want to save or discard the changes?</source>
-        <translation type="unfinished"></translation>
+        <translation>Вы хотите сохранить или отменить изменения?</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Do you want to cancel closing, save or discard the changes?</source>
-        <translation type="unfinished"></translation>
+        <translation>Вы хотите отменить закрытие, сохранить или отменить изменения?</translation>
     </message>
     <message>
         <location line="+4"/>
         <location line="+129"/>
         <location line="+115"/>
         <location line="+66"/>
         <location line="+22"/>
         <source>Octave Editor</source>
-        <translation type="unfinished"></translation>
+        <translation>Редактор Octave</translation>
     </message>
     <message>
         <location line="-331"/>
         <source>The file
 %1
 is about to be closed but has been modified.
 %2</source>
-        <translation type="unfinished"></translation>
+        <translation>Файл
+%1
+должен быть закрыть, но он был изменен.
+%2</translation>
     </message>
     <message>
         <location line="+210"/>
         <source>Octave Files (*.m);;All Files (*)</source>
-        <translation type="unfinished"></translation>
+        <translation>Файлы Octave (*.m);;Все файлы (*)</translation>
     </message>
     <message>
         <location line="+34"/>
         <source>File not saved! The selected file name
 %1
 is the same as the current file name</source>
-        <translation type="unfinished"></translation>
+        <translation>Файл не сохранен! Выбранное имя файла
+%1
+такое же, как и название текущего файла</translation>
     </message>
     <message>
         <location line="+81"/>
         <source>
 
 Warning: The contents in the editor is modified!</source>
-        <translation type="unfinished"></translation>
+        <translation>
+
+Предупреждение: содержимое в редакторе было изменено!</translation>
     </message>
     <message>
         <location line="+7"/>
         <source>It seems that the file
 %1
 has been deleted or renamed. Do you want to save it now?%2</source>
-        <translation type="unfinished"></translation>
+        <translation>Кажется, файл
+%1
+был удален или переименован. Вы хотите сохранить его?%2</translation>
     </message>
     <message>
         <location line="-203"/>
         <source>Could not open file %1 for write:
 %2.</source>
-        <translation type="unfinished"></translation>
+        <translation>Не удалось открыть файл %1 для записи:
+%2.</translation>
     </message>
     <message>
         <location line="+181"/>
         <source>It seems that &apos;%1&apos; has been modified by another application. Do you want to reload it?</source>
-        <translation type="unfinished"></translation>
+        <translation>Кажется, файл &apos;%1&apos; был изменен другим приложением. Вы хотите обновить его содержимое?</translation>
     </message>
 </context>
 <context>
     <name>files_dock_widget</name>
     <message>
         <location filename="../src/files-dock-widget.cc" line="+67"/>
         <source>File Browser</source>
-        <translation type="unfinished">Файловый менеджер</translation>
+        <translation>Файловый менеджер</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Browse your files.</source>
-        <translation type="unfinished">Просмотр файлов.</translation>
+        <translation>Просмотр файлов.</translation>
     </message>
     <message>
         <location line="+18"/>
         <source>Enter the path or filename</source>
-        <translation type="unfinished"></translation>
+        <translation>Введите путь или имя файла</translation>
     </message>
     <message>
         <location line="+10"/>
         <source>Move up one directory</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Show octave directory</source>
-        <translation type="unfinished"></translation>
+        <translation>Показывать директорию Octave</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Goto current octave directory</source>
-        <translation type="unfinished"></translation>
+        <translation>Перейти к текущей директории Octave</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Set octave directory</source>
-        <translation type="unfinished"></translation>
+        <translation>Установить директорию Octave</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Set octave directroy to current browser directory</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Actions on current directory</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Show Home directory</source>
-        <translation type="unfinished"></translation>
+        <translation>Показывать домашнюю директорию</translation>
     </message>
     <message>
         <location line="+10"/>
         <source>Search directory</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+3"/>
         <location line="+244"/>
         <source>Find Files ...</source>
-        <translation type="unfinished"></translation>
+        <translation>Найти файлы...</translation>
     </message>
     <message>
         <location line="-240"/>
         <location line="+252"/>
         <source>New File</source>
-        <translation type="unfinished"></translation>
+        <translation>Новый файл</translation>
     </message>
     <message>
         <location line="-249"/>
         <location line="+252"/>
         <source>New Directory</source>
-        <translation type="unfinished"></translation>
+        <translation>Новая директория</translation>
     </message>
     <message>
         <location line="-223"/>
         <source>Doubleclick a file to open it</source>
-        <translation type="unfinished"></translation>
+        <translation>Двойной щелчок по файлу для открытия</translation>
     </message>
     <message>
         <location line="+185"/>
         <source>Open</source>
-        <translation type="unfinished"></translation>
+        <translation>Открыть</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Open in Default Application</source>
-        <translation type="unfinished"></translation>
+        <translation>Открыть в приложении по умолчанию</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Copy Selection to Clipboard</source>
-        <translation type="unfinished"></translation>
+        <translation>Скопировать выделенное в буфер обмена</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Run</source>
-        <translation type="unfinished"></translation>
+        <translation>Запуск</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Load Data</source>
-        <translation type="unfinished"></translation>
+        <translation>Загрузить данные</translation>
     </message>
     <message>
         <location line="+6"/>
         <source>Set Current Directory</source>
-        <translation type="unfinished"></translation>
+        <translation>Установить текущую директорию</translation>
     </message>
     <message>
         <location line="+7"/>
         <source>Rename</source>
-        <translation type="unfinished"></translation>
+        <translation>Переименовать</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Delete</source>
-        <translation type="unfinished"></translation>
+        <translation>Удалить</translation>
     </message>
     <message>
         <location line="+107"/>
         <source>Rename file/directory</source>
-        <translation type="unfinished"></translation>
+        <translation>Переименовать файл/директорию</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Rename file/directory:
 </source>
-        <translation type="unfinished"></translation>
+        <translation>Переименовать файл/директорию:</translation>
     </message>
     <message>
         <location line="+0"/>
         <source>
  to: </source>
-        <translation type="unfinished"></translation>
+        <translation> в: </translation>
     </message>
     <message>
         <location line="+25"/>
         <location line="+11"/>
         <source>Delete file/directory</source>
-        <translation type="unfinished"></translation>
+        <translation>Удалить файл/директорию</translation>
     </message>
     <message>
         <location line="-10"/>
         <source>Are you sure you want to delete
 </source>
-        <translation type="unfinished"></translation>
+        <translation>Вы уверены, что хотите удалить</translation>
     </message>
     <message>
         <location line="+11"/>
         <source>Can not delete a directory that is not empty</source>
-        <translation type="unfinished"></translation>
+        <translation>Невозможно удалить непустую директорию</translation>
     </message>
     <message>
         <location line="+131"/>
         <source>Set directory of file browser</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+28"/>
         <source>Create File</source>
-        <translation type="unfinished"></translation>
+        <translation>Создать файл</translation>
     </message>
     <message>
         <location line="+0"/>
         <source>Create file in
 </source>
-        <translation type="unfinished"></translation>
+        <translation>Создать файл в</translation>
     </message>
     <message>
         <location line="+17"/>
         <source>Create Directory</source>
-        <translation type="unfinished"></translation>
+        <translation>Создать директорию</translation>
     </message>
     <message>
         <location line="+0"/>
         <source>Create folder in
 </source>
-        <translation type="unfinished"></translation>
+        <translation>Создать папку в</translation>
     </message>
 </context>
 <context>
     <name>find_dialog</name>
     <message>
         <location filename="../src/m-editor/find-dialog.cc" line="+77"/>
         <source>Find &amp;what:</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Найти:</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Re&amp;place with:</source>
-        <translation type="unfinished"></translation>
+        <translation>За&amp;менить на:</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Match &amp;case</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Учитывать регистр</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Search from &amp;start</source>
-        <translation type="unfinished"></translation>
+        <translation>И&amp;скать с начала</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>&amp;Wrap while searching</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+2"/>
         <source>&amp;Find Next</source>
-        <translation type="unfinished"></translation>
+        <translation>Найти &amp;следующее</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Find &amp;Previous</source>
-        <translation type="unfinished"></translation>
+        <translation>Найти п&amp;редыдущее</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>&amp;Replace</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Заменить</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Replace &amp;All</source>
-        <translation type="unfinished"></translation>
+        <translation>Заменить &amp;все</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>&amp;More</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Больше</translation>
     </message>
     <message>
         <location line="+13"/>
         <source>&amp;Whole words</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Regular E&amp;xpressions</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Регулярные выражения</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Search &amp;backward</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Искать в обратном направлении</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Search se&amp;lection</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+71"/>
         <source>Search from end</source>
-        <translation type="unfinished"></translation>
+        <translation>Искать с конца</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Search from start</source>
-        <translation type="unfinished"></translation>
+        <translation>Искать с начала</translation>
     </message>
     <message>
         <location line="+117"/>
         <source>Replace Result</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+1"/>
@@ -705,25 +723,25 @@ has been deleted or renamed. Do you want
     <message>
         <location line="+10"/>
         <source>Find Result</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+1"/>
         <source>No more matches found</source>
-        <translation type="unfinished"></translation>
+        <translation>Больше совпадений не найдено</translation>
     </message>
 </context>
 <context>
     <name>find_files_dialog</name>
     <message>
         <location filename="../src/find-files-dialog.cc" line="+47"/>
         <source>Find Files</source>
-        <translation type="unfinished"></translation>
+        <translation>Найти файлы</translation>
     </message>
     <message>
         <location line="+10"/>
         <source>Named:</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+2"/>
@@ -753,17 +771,17 @@ has been deleted or renamed. Do you want
     <message>
         <location line="+3"/>
         <source>Recurse directories</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Search recursively through directories for matching files</source>
-        <translation type="unfinished"></translation>
+        <translation>Искать рекурсивно в директориях подходящие файлы</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Include directories</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+2"/>
@@ -778,17 +796,17 @@ has been deleted or renamed. Do you want
     <message>
         <location line="+2"/>
         <source>Set matching name is case insensitive</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Contains text:</source>
-        <translation type="unfinished"></translation>
+        <translation>Содержит текст:</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Search must match text</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+4"/>
@@ -813,177 +831,177 @@ has been deleted or renamed. Do you want
     <message>
         <location line="+14"/>
         <source>Idle.</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Find</source>
-        <translation type="unfinished"></translation>
+        <translation>Найти</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Start search for matching files</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Stop</source>
-        <translation type="unfinished"></translation>
+        <translation>Завершить</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Stop searching</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+15"/>
         <source>File name/location</source>
-        <translation type="unfinished"></translation>
+        <translation>Имя/местоположение файла</translation>
     </message>
     <message>
         <location line="+17"/>
         <source>File contents</source>
-        <translation type="unfinished"></translation>
+        <translation>Содержимое файла</translation>
     </message>
     <message>
         <location line="+99"/>
         <source>Searching...</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+32"/>
         <source>Set search directory</source>
-        <translation type="unfinished"></translation>
+        <translation>Установить директорию для поиска</translation>
     </message>
 </context>
 <context>
     <name>find_files_model</name>
     <message>
         <location filename="../src/find-files-model.cc" line="+76"/>
         <source>Filename</source>
-        <translation type="unfinished"></translation>
+        <translation>Имя файла</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Directory</source>
-        <translation type="unfinished"></translation>
+        <translation>Директория</translation>
     </message>
 </context>
 <context>
     <name>history_dock_widget</name>
     <message>
         <location filename="../src/history-dock-widget.cc" line="+43"/>
         <source>Browse and search the command history.</source>
-        <translation type="unfinished">Просмотр и поиск в журнале выполненных команд.</translation>
+        <translation>Просмотр и поиск в журнале выполненных команд.</translation>
     </message>
     <message>
         <location line="+23"/>
         <source>Doubleclick a command to transfer it to the terminal.</source>
-        <translation type="unfinished">Двойной щелчок по команде перенесёт её в командную строку.</translation>
+        <translation>Двойной щелчок по команде перенесёт её в командную строку.</translation>
     </message>
     <message>
         <location line="+6"/>
         <source>Enter text to filter the command history.</source>
-        <translation type="unfinished">Введите текст для фильтрации выполненных команд.</translation>
+        <translation>Введите текст для фильтрации выполненных команд.</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Command History</source>
-        <translation type="unfinished">Журнал выполненных команд</translation>
+        <translation>Журнал выполненных команд</translation>
     </message>
     <message>
         <location line="+20"/>
         <source>Copy</source>
-        <translation type="unfinished"></translation>
+        <translation>Копировать</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Evaluate</source>
-        <translation type="unfinished"></translation>
+        <translation>Выполнить</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Create script</source>
-        <translation type="unfinished"></translation>
+        <translation>Создать скрипт</translation>
     </message>
 </context>
 <context>
     <name>main_window</name>
     <message>
         <location filename="../src/main-window.cc" line="+163"/>
         <source>Load Workspace</source>
-        <translation type="unfinished">Загрузить область переменных</translation>
+        <translation>Загрузить область переменных</translation>
     </message>
     <message>
         <location line="+435"/>
         <location line="+847"/>
         <source>About Octave</source>
-        <translation type="unfinished">Об Octave</translation>
+        <translation>Об Octave</translation>
     </message>
     <message>
         <location line="-369"/>
         <source>&amp;File</source>
-        <translation type="unfinished">&amp;Файл</translation>
+        <translation>&amp;Файл</translation>
     </message>
     <message>
         <location line="+54"/>
         <source>New</source>
-        <translation type="unfinished"></translation>
+        <translation>Новый</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Script</source>
-        <translation type="unfinished"></translation>
+        <translation>Скрипт</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Function</source>
-        <translation type="unfinished"></translation>
+        <translation>Функция</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Figure</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="-58"/>
         <source>Open...</source>
-        <translation type="unfinished"></translation>
+        <translation>Открыть...</translation>
     </message>
     <message>
         <location line="+20"/>
         <source>Preferences...</source>
-        <translation type="unfinished"></translation>
+        <translation>Настройки...</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Exit</source>
-        <translation type="unfinished">Выход</translation>
+        <translation>Выход</translation>
     </message>
     <message>
         <location line="+52"/>
         <source>&amp;Edit</source>
-        <translation type="unfinished">&amp;Правка</translation>
+        <translation>&amp;Правка</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Undo</source>
-        <translation type="unfinished"></translation>
+        <translation>Отменить</translation>
     </message>
     <message>
         <location line="+7"/>
         <source>Copy</source>
-        <translation type="unfinished"></translation>
+        <translation>Копировать</translation>
     </message>
     <message>
         <location line="+6"/>
         <source>Paste</source>
-        <translation type="unfinished"></translation>
+        <translation>Вставить</translation>
     </message>
     <message>
         <location line="-1026"/>
         <location line="+946"/>
         <source>Save Workspace As</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
@@ -1009,250 +1027,250 @@ has been deleted or renamed. Do you want
     <message>
         <location line="+736"/>
         <source>Clear Clipboard</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Find Files...</source>
-        <translation type="unfinished"></translation>
+        <translation>Найти файлы...</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Clear Command Window</source>
-        <translation type="unfinished"></translation>
+        <translation>Очистить командное окно</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Clear Command History</source>
-        <translation type="unfinished"></translation>
+        <translation>Очистить журнал выполненных команд</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Clear Workspace</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+40"/>
         <source>De&amp;bug</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Отладка</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Step</source>
-        <translation type="unfinished"></translation>
+        <translation>Шаг</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Step in</source>
-        <translation type="unfinished"></translation>
+        <translation>Шаг вперед</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Step out</source>
-        <translation type="unfinished"></translation>
+        <translation>Шаг назад</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Continue</source>
-        <translation type="unfinished"></translation>
+        <translation>Продолжить</translation>
     </message>
     <message>
         <location line="+8"/>
         <source>Exit Debug Mode</source>
-        <translation type="unfinished"></translation>
+        <translation>Выйти из режима отладки</translation>
     </message>
     <message>
         <location line="+48"/>
         <source>Show File Browser</source>
-        <translation type="unfinished"></translation>
+        <translation>Показывать файловый менеджер</translation>
     </message>
     <message>
         <location line="+12"/>
         <source>Show News Window</source>
-        <translation type="unfinished"></translation>
+        <translation>Показывать окно новостей</translation>
     </message>
     <message>
         <location line="+11"/>
         <source>File Browser</source>
-        <translation type="unfinished">Файловый менеджер</translation>
+        <translation>Файловый менеджер</translation>
     </message>
     <message>
         <location line="+12"/>
         <source>News</source>
-        <translation type="unfinished"></translation>
+        <translation>Новости</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Reset Default Window Layout</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+86"/>
         <source>Octave Packages</source>
-        <translation type="unfinished"></translation>
+        <translation>Пакеты Octave</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Share Code</source>
-        <translation type="unfinished"></translation>
+        <translation>Поделиться кодом</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Contribute to Octave</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Octave Developer Resources</source>
-        <translation type="unfinished"></translation>
+        <translation>Ресурсы разработчика Octave</translation>
     </message>
     <message>
         <location line="+32"/>
         <source>On Disk</source>
-        <translation type="unfinished"></translation>
+        <translation>На диске</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Online</source>
-        <translation type="unfinished"></translation>
+        <translation>Онлайн</translation>
     </message>
     <message>
         <location line="+12"/>
         <source>&amp;News</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Новости</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Release Notes</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Community News</source>
-        <translation type="unfinished"></translation>
+        <translation>Новости сообщества</translation>
     </message>
     <message>
         <location line="+30"/>
         <source>Enter directory name</source>
-        <translation type="unfinished"></translation>
+        <translation>Введите название директории</translation>
     </message>
     <message>
         <location line="+8"/>
         <source>Current Directory: </source>
-        <translation type="unfinished"></translation>
+        <translation>Текущая директория: </translation>
     </message>
     <message>
         <location line="+4"/>
         <source>One directory up</source>
-        <translation type="unfinished"></translation>
+        <translation>Одной директорией вверх</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Browse directories</source>
-        <translation type="unfinished"></translation>
+        <translation>Просмотр директорий</translation>
     </message>
     <message>
         <location line="-445"/>
         <source>Load workspace</source>
-        <translation type="unfinished"></translation>
+        <translation>Загрузить область переменных</translation>
     </message>
     <message>
         <location line="+200"/>
         <source>&amp;Window</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Окно</translation>
     </message>
     <message>
         <location line="+6"/>
         <source>Show Command Window</source>
-        <translation type="unfinished"></translation>
+        <translation>Показывать командное окно</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Show Command History</source>
-        <translation type="unfinished"></translation>
+        <translation>Показывать журнал выполненных команд</translation>
     </message>
     <message>
         <location line="+6"/>
         <source>Show Workspace</source>
-        <translation type="unfinished"></translation>
+        <translation>Показывать область переменных</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Show Editor</source>
-        <translation type="unfinished"></translation>
+        <translation>Показывать редактор</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Show Documentation</source>
-        <translation type="unfinished"></translation>
+        <translation>Показывать документацию</translation>
     </message>
     <message>
         <location line="+8"/>
         <source>Command Window</source>
-        <translation type="unfinished"></translation>
+        <translation>Командное окно</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Command History</source>
-        <translation type="unfinished">Журнал выполненных команд</translation>
+        <translation>Журнал выполненных команд</translation>
     </message>
     <message>
         <location line="+6"/>
         <source>Workspace</source>
-        <translation type="unfinished">Область переменных</translation>
+        <translation>Область переменных</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Editor</source>
-        <translation type="unfinished">Редактор</translation>
+        <translation>Редактор</translation>
     </message>
     <message>
         <location line="+3"/>
         <location line="+132"/>
         <source>Documentation</source>
-        <translation type="unfinished">Документация</translation>
+        <translation>Документация</translation>
     </message>
     <message>
         <location line="-48"/>
         <source>&amp;Help</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Помощь</translation>
     </message>
     <message>
         <location line="+7"/>
         <source>Report Bug</source>
-        <translation type="unfinished">Сообщить об ошибке</translation>
+        <translation>Сообщить об ошибке</translation>
     </message>
 </context>
 <context>
     <name>news_dock_widget</name>
     <message>
         <location filename="../src/news-dock-widget.cc" line="+44"/>
         <source>Community News</source>
-        <translation type="unfinished"></translation>
+        <translation>Новости сообщества</translation>
     </message>
 </context>
 <context>
     <name>octave_dock_widget</name>
     <message>
         <location filename="../src/octave-dock-widget.cc" line="+56"/>
         <location line="+159"/>
         <source>Undock widget</source>
-        <translation type="unfinished"></translation>
+        <translation>Открепить виджет</translation>
     </message>
     <message>
         <location line="-149"/>
         <source>Hide widget</source>
-        <translation type="unfinished"></translation>
+        <translation>Спрятать виджет</translation>
     </message>
     <message>
         <location line="+104"/>
         <source>Dock widget</source>
-        <translation type="unfinished"></translation>
+        <translation>Закрепить виджет</translation>
     </message>
 </context>
 <context>
     <name>octave_qscintilla</name>
     <message>
         <location filename="../src/m-editor/octave-qscintilla.cc" line="+127"/>
         <source>Help on</source>
         <translation type="unfinished"></translation>
@@ -1260,113 +1278,113 @@ has been deleted or renamed. Do you want
     <message>
         <location line="+2"/>
         <source>Documentation on</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Edit</source>
-        <translation type="unfinished"></translation>
+        <translation>Редактировать</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>&amp;Run Selection</source>
         <translation type="unfinished"></translation>
     </message>
 </context>
 <context>
     <name>octave_qt_link</name>
     <message>
         <location filename="../src/octave-qt-link.cc" line="+270"/>
         <source>The file %1 does not exist in the load path.  To debug the function you are editing, you must either change to the directory %2 or add that directory to the load path.</source>
-        <translation type="unfinished"></translation>
+        <translation>Файл %1 не существует в пути загрузки.  Чтобы отладить редактируемую вами функцию, вы должны либо изменить директорию на %2, или добавить эту директорию в путь загрузки.</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>The file %1 is shadowed by a file with the same name in the load path.  To debug the function you are editing, change to the directory %2.</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Change Directory or Add Directory to Load Path</source>
-        <translation type="unfinished"></translation>
+        <translation>Изменить директорию или добавить директорию в путь загрузки</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Change Directory</source>
-        <translation type="unfinished"></translation>
+        <translation>Изменить директорию</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Add Directory to Load Path</source>
-        <translation type="unfinished"></translation>
+        <translation>Добавить директорию в путь загрузки</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Cancel</source>
-        <translation type="unfinished"></translation>
+        <translation>Отмена</translation>
     </message>
 </context>
 <context>
     <name>settings_dialog</name>
     <message>
         <location filename="../src/settings-dialog.ui" line="+29"/>
         <source>Settings</source>
-        <translation type="unfinished">Параметры</translation>
+        <translation>Параметры</translation>
     </message>
     <message>
         <location line="+13"/>
         <source>General</source>
-        <translation type="unfinished"></translation>
+        <translation>Основное</translation>
     </message>
     <message>
         <location line="+97"/>
         <source>Octave logo only</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Только логотип Octave</translation>
     </message>
     <message>
         <location line="+10"/>
         <source>Letter icons</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+46"/>
         <source>Editor</source>
-        <translation type="unfinished">Редактор</translation>
+        <translation>Редактор</translation>
     </message>
     <message>
         <location line="+16"/>
         <source>Show white space</source>
-        <translation type="unfinished"></translation>
+        <translation>Показывать пробелы</translation>
     </message>
     <message>
         <location line="+27"/>
         <source>Do not show white spaces used for indentation</source>
-        <translation type="unfinished"></translation>
+        <translation>Не показывать пробелы, используемые для отступов</translation>
     </message>
     <message>
         <location line="+28"/>
         <source>Color</source>
-        <translation type="unfinished"></translation>
+        <translation>Цвет</translation>
     </message>
     <message>
         <location line="+120"/>
         <source>Indent width</source>
-        <translation type="unfinished"></translation>
+        <translation>Ширина отступа</translation>
     </message>
     <message>
         <location line="+7"/>
         <source>Tab indents line</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+7"/>
         <source>Auto indentation</source>
-        <translation type="unfinished"></translation>
+        <translation>Авто-отступы</translation>
     </message>
     <message>
         <location line="+20"/>
         <source>Tab width</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+7"/>
@@ -1386,32 +1404,32 @@ has been deleted or renamed. Do you want
     <message>
         <location line="+71"/>
         <source>Match keywords</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+13"/>
         <source>Case sensitive</source>
-        <translation type="unfinished"></translation>
+        <translation>Чувствительный к регистру</translation>
     </message>
     <message>
         <location line="+13"/>
         <source>Replace word by suggested one</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+23"/>
         <source>Match words in document</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+61"/>
         <source>Restore editor tabs from previous session on startup</source>
-        <translation type="unfinished"></translation>
+        <translation>Восстанавливать вкладки редактора с предыдущей сессии во время запуска</translation>
     </message>
     <message>
         <location line="+13"/>
         <source>Create non existing files without prompting</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+47"/>
@@ -1426,187 +1444,187 @@ has been deleted or renamed. Do you want
     <message>
         <location line="+22"/>
         <source>Editor Styles</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+24"/>
         <source>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select font, font size (as difference to the default size), font decoration (bold, italic, underline), textcolor and background color (for the latter, the color pink (255,0,255) is a placeholder for the default background color)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</source>
-        <translation type="unfinished"></translation>
+        <translation>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Выберите шрифт, размер шрифта (отличный от размера по умолчанию), стиль шрифта (жирный, курсив, подчеркнутый),цвет текста и цвет заднего фона (для последнего розовый цвет (255,0,255) используется по умолчанию)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</translation>
     </message>
     <message>
         <location line="+76"/>
         <source>Use Foreground Color</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+32"/>
         <source>Terminal Colors</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+45"/>
         <source>Font</source>
-        <translation type="unfinished"></translation>
+        <translation>Шрифт</translation>
     </message>
     <message>
         <location line="-757"/>
         <source>Show line numbers</source>
-        <translation type="unfinished"></translation>
+        <translation>Показывать количество строк</translation>
     </message>
     <message>
         <location line="+27"/>
         <source>Highlight current line</source>
-        <translation type="unfinished"></translation>
+        <translation>Подсвечивать текущую строку</translation>
     </message>
     <message>
         <location line="+262"/>
         <source>Code completion</source>
-        <translation type="unfinished"></translation>
+        <translation>Дополнение кода</translation>
     </message>
     <message>
         <location line="-282"/>
         <source>Show complete path in window title</source>
-        <translation type="unfinished"></translation>
+        <translation>Показывать полный путь в заголовке окна</translation>
     </message>
     <message>
         <location line="-72"/>
         <source>Graphic icons</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+633"/>
         <source>emacs</source>
-        <translation type="unfinished">emacs</translation>
+        <translation>emacs</translation>
     </message>
     <message>
         <location line="+67"/>
         <source>Terminal</source>
-        <translation type="unfinished">Командная строка</translation>
+        <translation>Командная строка</translation>
     </message>
     <message>
         <location line="+15"/>
         <source>Cursor type:</source>
-        <translation type="unfinished"></translation>
+        <translation>Тип курсора:</translation>
     </message>
     <message>
         <location line="+23"/>
         <source>Cursor blinking</source>
-        <translation type="unfinished"></translation>
+        <translation>Мигающий курсор</translation>
     </message>
     <message>
         <location line="+101"/>
         <source>Font size</source>
-        <translation type="unfinished"></translation>
+        <translation>Размер шрифта</translation>
     </message>
     <message>
         <location line="+35"/>
         <source>File Browser</source>
-        <translation type="unfinished">Файловый менеджер</translation>
+        <translation>Файловый менеджер</translation>
     </message>
     <message>
         <location line="+6"/>
         <source>Show file size</source>
-        <translation type="unfinished">Показывать размер файлов</translation>
+        <translation>Показывать размер файлов</translation>
     </message>
     <message>
         <location line="+7"/>
         <source>Show file type</source>
-        <translation type="unfinished">Показывать типы файлов</translation>
+        <translation>Показывать типы файлов</translation>
     </message>
     <message>
         <location line="+7"/>
         <source>Show date of last modification</source>
-        <translation type="unfinished">Показывать дату последнего изменения</translation>
+        <translation>Показывать дату последнего изменения</translation>
     </message>
     <message>
         <location line="+7"/>
         <source>Show hidden files</source>
-        <translation type="unfinished">Показывать скрытые файлы</translation>
+        <translation>Показывать скрытые файлы</translation>
     </message>
     <message>
         <location line="+7"/>
         <source>Synchronize octave directory with the file browser</source>
-        <translation type="unfinished"></translation>
+        <translation>Синхронизировать директорию octave с файловым менеджером</translation>
     </message>
     <message>
         <location line="+7"/>
         <source>Alternating row colors</source>
-        <translation type="unfinished">Чередующиеся цвета строк</translation>
+        <translation>Чередующиеся цвета строк</translation>
     </message>
     <message>
         <location line="+21"/>
         <source>Workspace</source>
-        <translation type="unfinished">Область переменных</translation>
+        <translation>Область переменных</translation>
     </message>
     <message>
         <location line="+30"/>
         <source>Storage Class Colors</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+35"/>
         <source>Network</source>
-        <translation type="unfinished"></translation>
+        <translation>Сеть</translation>
     </message>
     <message>
         <location line="+45"/>
         <source>Use proxy server</source>
-        <translation type="unfinished"></translation>
+        <translation>Использовать прокси-сервер</translation>
     </message>
     <message>
         <location line="+10"/>
         <source>Proxy Type:</source>
-        <translation type="unfinished"></translation>
+        <translation>Тип прокси:</translation>
     </message>
     <message>
         <location line="-33"/>
         <source>HttpProxy</source>
-        <translation type="unfinished"></translation>
+        <translation>HttpProxy</translation>
     </message>
     <message>
         <location line="-1120"/>
         <source>Icon set for dock widgets</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+7"/>
         <source>Language (requires restart)</source>
-        <translation type="unfinished"></translation>
+        <translation>Язык (требуется перезапуск)</translation>
     </message>
     <message>
         <location line="+7"/>
         <source>Icon size</source>
-        <translation type="unfinished"></translation>
+        <translation>Размер иконок</translation>
     </message>
     <message>
         <location line="+1111"/>
         <source>Socks5Proxy</source>
-        <translation type="unfinished"></translation>
+        <translation>Socks5Proxy</translation>
     </message>
     <message>
         <location line="-16"/>
         <source>Hostname:</source>
-        <translation type="unfinished"></translation>
+        <translation>Имя хоста:</translation>
     </message>
     <message>
         <location line="+54"/>
         <source>Port:</source>
-        <translation type="unfinished"></translation>
+        <translation>Порт:</translation>
     </message>
     <message>
         <location line="-27"/>
         <source>Username:</source>
-        <translation type="unfinished"></translation>
+        <translation>Имя пользователя:</translation>
     </message>
     <message>
         <location line="+37"/>
         <source>Password:</source>
-        <translation type="unfinished">Пароль:</translation>
+        <translation>Пароль:</translation>
     </message>
     <message>
         <location filename="../src/settings-dialog.cc" line="+70"/>
         <location line="+4"/>
         <location line="+346"/>
         <source>System setting</source>
         <translation type="unfinished"></translation>
     </message>
@@ -1623,168 +1641,179 @@ has been deleted or renamed. Do you want
     <message>
         <location line="+1"/>
         <source>Underline Cursor</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+140"/>
         <source>Difference to the default size</source>
-        <translation type="unfinished"></translation>
+        <translation>Отличие то размера по умолчанию</translation>
     </message>
     <message>
         <location line="+6"/>
         <source>Background color, pink (255,0,255) means default</source>
-        <translation type="unfinished"></translation>
+        <translation>Цвет заднего фона, розовый (255,0,255) по умолчанию</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>b</source>
-        <translation type="unfinished"></translation>
+        <translation>b</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>i</source>
-        <translation type="unfinished"></translation>
+        <translation>i</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>u</source>
-        <translation type="unfinished"></translation>
+        <translation>u</translation>
     </message>
 </context>
 <context>
     <name>terminal_dock_widget</name>
     <message>
         <location filename="../src/terminal-dock-widget.cc" line="+38"/>
         <source>Command Window</source>
-        <translation type="unfinished"></translation>
+        <translation>Командное окно</translation>
     </message>
 </context>
 <context>
     <name>webinfo</name>
     <message>
         <location filename="../src/qtinfo/webinfo.cc" line="+79"/>
         <source>Type here and press &apos;Return&apos; to search</source>
-        <translation type="unfinished"></translation>
+        <translation>Вводите сюда и нажмите &apos;Enter&apos;, чтобы начать поиск</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Global search</source>
-        <translation type="unfinished"></translation>
+        <translation>Глобальный поиск</translation>
     </message>
 </context>
 <context>
     <name>welcome_wizard</name>
     <message>
         <location filename="../src/welcome-wizard.cc" line="+40"/>
         <source>Welcome to GNU Octave</source>
-        <translation type="unfinished"></translation>
+        <translation>Добро пожаловать в GNU Octave</translation>
     </message>
     <message>
         <location line="+14"/>
         <source>Welcome to Octave!</source>
-        <translation type="unfinished"></translation>
+        <translation>Добро пожаловать в Octave!</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>You seem to be using the Octave graphical interface for the first  time on this computer.  Click &apos;Finish&apos; to write a configuration file  and launch Octave GUI.</source>
-        <translation type="unfinished"></translation>
+        <translation>Кажется, вы используете графический интерфейс для Octave впервые на этом компьютере. Щелкните на кнопку &apos;Завершить&apos;, чтобы записать конфигурационный файл и запустить интерфейс для Octave.</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>The configuration file is stored in %1. If that file exists, you will not see this dialog when Octave starts again.</source>
-        <translation type="unfinished"></translation>
+        <translation>Конфигурационный файл хранится в %1. Если этот файл существует, вы больше не увидите это всплывающее окно при запуске Octave.</translation>
     </message>
     <message>
         <location line="+23"/>
         <source>&lt;html&gt;&lt;head&gt;
 &lt;style&gt;
 a:link { text-decoration: underline; color: #0000ff; }
 &lt;/style&gt;
 &lt;head/&gt;&lt;body&gt;
 &lt;p&gt;For more information about Octave:&lt;/p&gt;
 &lt;ul&gt;
 &lt;li&gt;Visit &lt;a href=&quot;http://octave.org&quot;&gt;http://octave.org&lt;/a&gt;&lt;/li&gt;
 &lt;li&gt;Get the documentation online as &lt;a href=&quot;http://www.gnu.org/software/octave/doc/interpreter/index.html&quot;&gt;html&lt;/a&gt;- or &lt;a href=&quot;http://www.gnu.org/software/octave/octave.pdf&quot;&gt;pdf&lt;/span&gt;&lt;/a&gt;-document&lt;/li&gt;
 &lt;li&gt;Open the documentation browser of Octave GUI with the help menu&lt;/li&gt;
 &lt;/ul&gt;
 &lt;/body&gt;&lt;/html&gt;</source>
-        <translation type="unfinished"></translation>
+        <translation>&lt;html&gt;&lt;head&gt;
+&lt;style&gt;
+a:link { text-decoration: underline; color: #0000ff; }
+&lt;/style&gt;
+&lt;head/&gt;&lt;body&gt;
+&lt;p&gt;Для получения дополнительной информации об Octave:&lt;/p&gt;
+&lt;ul&gt;
+&lt;li&gt;Посетите &lt;a href=&quot;http://octave.org&quot;&gt;http://octave.org&lt;/a&gt;&lt;/li&gt;
+&lt;li&gt;Получите документацию онлайн в виде&lt;a href=&quot;http://www.gnu.org/software/octave/doc/interpreter/index.html&quot;&gt;html&lt;/a&gt;- или &lt;a href=&quot;http://www.gnu.org/software/octave/octave.pdf&quot;&gt;pdf&lt;/span&gt;&lt;/a&gt;-документа&lt;/li&gt;
+&lt;li&gt;Откройте просмотрщик документации через меню &quot;Помощь&quot;&lt;/li&gt;
+&lt;/ul&gt;
+&lt;/body&gt;&lt;/html&gt;</translation>
     </message>
     <message>
         <location line="+20"/>
         <source>Finish</source>
-        <translation type="unfinished"></translation>
+        <translation>Завершить</translation>
     </message>
 </context>
 <context>
     <name>workspace_model</name>
     <message>
         <location filename="../src/workspace-model.cc" line="-42"/>
         <source>Name</source>
-        <translation type="unfinished">Идентификатор</translation>
+        <translation>Идентификатор</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Class</source>
-        <translation type="unfinished"></translation>
+        <translation>Тип</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Dimension</source>
-        <translation type="unfinished"></translation>
+        <translation>Размерность</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Value</source>
-        <translation type="unfinished">Значение</translation>
+        <translation>Значение</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Storage Class</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+107"/>
         <source>Right click to copy, rename, or display</source>
-        <translation type="unfinished"></translation>
+        <translation>Правый щелчок для копирования, переименования или отображения</translation>
     </message>
 </context>
 <context>
     <name>workspace_view</name>
     <message>
         <location filename="../src/workspace-view.cc" line="+47"/>
         <source>Workspace</source>
-        <translation type="unfinished">Область переменных</translation>
+        <translation>Область переменных</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>View the variables in the active workspace.</source>
-        <translation type="unfinished">Просмотр содержимого текущей области переменных.</translation>
+        <translation>Просмотр содержимого текущей области переменных.</translation>
     </message>
     <message>
         <location line="+75"/>
         <source>Copy</source>
-        <translation type="unfinished"></translation>
+        <translation>Копировать</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Rename</source>
-        <translation type="unfinished"></translation>
+        <translation>Переименовать</translation>
     </message>
     <message>
         <location line="+8"/>
         <source>Only top-level symbols may be renamed.</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+125"/>
         <source>View the variables in the active workspace.&lt;br&gt;</source>
-        <translation type="unfinished"></translation>
+        <translation>Просмотр содержимого текущей области переменных.&lt;br&gt; </translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Colors for the storage class:</source>
         <translation type="unfinished"></translation>
     </message>
 </context>
 </TS>
